docker/envoy-gateway/Dockerfile

```diff
@@ -1,8 +1,7 @@
 # Use distroless as minimal base image to package the manager binary
```

Comment:
prefer keeping this file at the top level for now, so its simpler to remember things
* top level Makefile, Dockerfile
* rest of the helpers live under `tools/`

Reply:
this could also be moved to `tools/docker/`

Reply:
Got it, thanks @arkodg